topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,642 @@
+
+The RTK/RAS/PI3K molecular genetic axis controls critical cellular functions and is commonly altered in various cancers ({% cite Fruman2014 %}). Perturbations across this axis can lead to deficiencies in cell-cycle, survival, metabolism, motility and genome stability, triggering hallmark phenotypes of cancer. The constitutive activation and presence of phosphatidylinositol-3,4,5-trisphosphate (PIP3) trigger membrane-bound onco-signalosomes. This presents significant challenges for treatment, as PI3K cascade can be activated in several ways ({% cite Zhao2006 %}).
+
+Cancer driver genes comprising both oncogenes (OG) and Tumor suppressor genes (TSG) share common phenotypical outcome. However, they often have divergent molecular mechanisms  that drive the outcome. We are interested in capturing mutational specific differential transcriptional outcome among OG and TSG. In Figure 1, genes in red are oncogenes (have activating or copy gain) and genes in blue are tumor suppressor genes (have inactivating or copy loss).
+
+In this tutorial we plan to measure aberrant PI3K pathway activity in TCGA dataset using RNA-Seq information and mutational and copy number information of following frequently altered genes. We named this tutorial as PanCancer Aberrant Pathway Activity Analysis (PAPAA).
+
+![pi3k_pathway](../../images/aberrant_pi3k_pathway_analysis/pi3k_pathway.png "Genes used in this study from ERK/RAS/PI3K pathway. Red text indicates Oncogenes (OG), blue text indicates Tumor suppressors genes (TSG).")
+
+> ### Agenda
+>
+> Let's learn how to install this tutorial using Docker with galaxy, and how to download the datasets from Zenodo :
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# **Pre-installed tutorial tools, datasets and workflows from the docker image**
+> An efficient way to install and run the tutorial using papaa tools is available on docker based galaxy instance that has pre-installed papaa tool-suite as **papaa** under tools section. Additionally this local galaxy instance comes with datasets and workflow for generating PI3K_OG classifier. Instructions to run the docker image is below.
+
+> ### {% icon hands_on %} Hands-on: Tutorial for galaxy docker container installation and running the workflow:
+> 1. Pulling the docker image from docker hub: Open a terminal and type the following command: 
+>       ```docker pull nvk747/galaxy_papaa:0.1.9```
+> 2. For running the docker container use the following command :
+>       ```docker run -p 8888:80 --detach nvk747/galaxy_papaa:0.1.9```
+> 3. Open the following [*http://localhost:8888*](http://localhost:8888) in your web browser.
```

Comment:
I followed the steps mentioned above to pull the docker image but unable to see anything at http://localhost:8888
Do I miss anything?

![papaa_tutorial](https://user-images.githubusercontent.com/3022518/107407353-d9ed0900-6b09-11eb-9f68-94afd7294442.png)

Reply:
@anuprulez I see the docker running, did you still find a problem with http://localhost:8888?

Reply:
@nvk747 It works now. There was some other docker container running locally at the same port. Thanks!